### ISSUE TITLE
Docker Images / Kubernetes support for clustering?

### ISSUE THREAD

**Issue by loeken** [ant-media/Ant-Media-Server#2479]
**Is your feature request related to a problem? Please describe.**
Hello, been playing around with ant media server a bit lately, and also found the docker section in the documentation. however the provided example seems to have some problems ( at least for me ) I can get it to run but whenever i ./change_mode to clustered and feed it a mongo ip the application seems to crash on me. I was also unable to find any other published docker images.

As a developer or operations guy i would find it really useful to have some working docker images ( ideally some docker-compose file that defines an example layout for a small cluster ) I think this would also be useful for you guys - the easier it is for ppl to use your software the easier it is to test / use your software.

**Describe the solution you'd like**
- probably best to start by reworking the docker images and not start a service but rather some start script that takes care of sending logs back to docker (stdout/stderr).
- maybe extand this start script so you can pass on the parameters for clustering when starting up. then we dont have to restart the application inside the container..

- Update of the documentation
  - 1 docker-compose for non clustered usage
  - 1 docker-compose for clustered usage

**Describe alternatives you've considered**

My endgame is to run it in a self manager k8s cluster - but if i get working docker images i can finish the rest myself

**Additional context**
If it helps somebody else: https://github.com/loeken/ant-media-server

I'm basically mostly using the Dockerfile from the documentation but have removed the part where it runs the change to cluster script ( as whenever i run this the application/container dies for me )

If there is some interest to work on this i could also provide more details log of ant media crashing / or somebody can clone the repo and test around

**Comment by mekya** [comment]
Yeah you're right. The work you mention is almost finished at this issue #2390

**Comment by mekya** [comment]
We've a draft documentation for that. Check this out 
https://github.com/ant-media/Ant-Media-Server/wiki/Getting-Started-with-Ant-Media-Server-Kubernetes
